x-pack/solutions/security/plugins/security_solution/public/configurations/translations.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const BASIC_RULES = i18n.translate(
+  'xpack.securitySolution.configurations.tabs.basic_rules',
+  {
+    defaultMessage: 'Rules',
+  }
+);
+export const KNOWLEDGE_SOURCES = i18n.translate(
+  'xpack.securitySolution.configurations.tabs.knowledge_sources',
+  {
+    defaultMessage: 'Knowledge Sources',
```

Comment:
```suggestion
    defaultMessage: 'Knowledge sources',
```

The figma has sources lowercase

<img width="285" alt="Screenshot 2025-03-28 at 10 02 09 AM" src="https://github.com/user-attachments/assets/edfdf98c-df76-4bb0-bf11-b9c05da27d35" />